### PR TITLE
manifest: sdk-zephyr: apply modem cherry picks

### PR DESCRIPTION
**Pull request by tomi-font**
To get the generic modem driver support for nRF91 series (w/ SLM).

### CODE REVIEW

**Comment by PerMac** [comment]
@tomi-font  I think you've picked a wrong PR in the manifest. Description here doesn't match the modified entry.

**Comment by tomi-font** [comment]
@PerMac You are right. Thanks for the heads-up!

**Review by SeppoTakalo — APPROVED** [review]

**Review by MarkusLassila — APPROVED** [review]
Needed for Serial LTE Modem.

File: west.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 962eb7320fdb76ffe265d8bab7330ad472cb6f28
+      revision: pull/1503/head
```
Comment (jfischer-no) [inline west.yml:64]: Let's give it a chance.
revision: c5c7fb3967905dcaef1035ac8d5b234176a2cc0a
https://github.com/nrfconnect/sdk-zephyr/commit/c5c7fb3967905dcaef1035ac8d5b234176a2cc0a

**Comment by jfischer-no** [comment]
@tomi-font Sorry, #14006 is merged, zephyr-revision will be updated in https://github.com/nrfconnect/sdk-nrf/pull/13768

**Comment by jfischer-no** [comment]
https://github.com/nrfconnect/sdk-nrf/pull/13768 is merged

**Pull request closed without merging**